examples/seq2seq/README.md

```diff
@@ -160,11 +159,12 @@ python examples/seq2seq/run_seq2seq.py \
 And the same with MBart:
 
 ```bash
-python examples/seq2seq/run_seq2seq.py \
+python examples/seq2seq/run_trans.py \
     --model_name_or_path facebook/mbart-large-en-ro  \
     --do_train \
     --do_eval \
-    --task translation_en_to_ro \
+    --source_lang en_XX \
+    --target_lang ro_RO 
     --dataset_name wmt16 \
     --dataset_config_name ro-en \
     --source_lang en_XX \
```

Comment:
here where they are already there.

Reply:
hmm, weird I guess one can't delete lines in "suggestion" - I tried to delete those 2 lines.